imports/ui/dashboard/dashboardMc211.js

```diff
@@ -0,0 +1,63 @@
+import CollectionsCount from '/imports/api/collectionsCount/collectionsCount';
+import {
+  DefaultAdminAccessRoles,
+  PendingClientsAccessRoles,
+  ResponsesAccessRoles,
+} from '/imports/config/permissions';
+import './dashboardMc211.html';
+
+
+const allWidgets = [
+  {
```

Comment:
What do you think if we'll unify shared widgets into one file and later extend them with app-specific ones?

Reply:
Then you should create a kind of logic where each app can modify the widgets list, add new ones, reorder, delete. Also fetching widgets values should be delegated to backend, preferably method calls - each widget would have its method